scripts/commands/menu.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- func: menu
+-- desc: Shows a basic test menu with three options
+-----------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = ""
+}
+
+function onTrigger(player)
+    player:customMenu({ "Test Menu (Play Effect)",
```

Comment:
This styling kills me, and I'd hate to set the example for further users.

Reply:
Happy to accept other stylings, all Allman?

Reply:
I think we sort of cover this case:

https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#inline-tables

Reply:
I think I'd be OK adjusting this rule to be "define your table beforehand and pass it in", I never really liked this edge case